watertap/unit_models/translators/translator_asm2d_adm1.py

```diff
@@ -191,6 +190,15 @@ def build(self):
             mutable=True,
             doc="Smoothing factor",
         )
+        self.heaviside_k = Param(
+            initialize=1e1,
+            units=pyunits.dimensionless,
+            mutable=True,
+            doc="Smooth heaviside k parameter",
+        )
```

Comment:
Should this be a single value or can there be cases where we might need a separate k value for a separate smooth heaviside relationship?

Reply:
In this translator block, I think we can get away with one value for k. In the ASM1-ADM1, we may need multiple k values.